src/core/textures/BaseTexture.js

```diff
@@ -195,10 +214,79 @@ BaseTexture.prototype.update = function ()
 
     this.isPowerOfTwo = bitTwiddle.isPow2(this.realWidth) && bitTwiddle.isPow2(this.realHeight);
 
+    if (this.isSVG()) {
+        this.updateSVG();
+    }
+
     this.emit('update', this);
 };
 
 /**
+ * Helper to check if this texture is loaded from svg.
+ * @return {Boolean} True if source is an svg image
+ */
+BaseTexture.prototype.isSVG = function () {
+    return this.source.type && this.source.type === 'svg';
+};
+
+/**
+ * Updates texture from SVG image using canvas. The original SVG Image is stored
+ * as `origSource` and the created canvas is is now `source`. The SVG is scaled
+ * using `sourceScale`.
+ */
+BaseTexture.prototype.updateSVG = function ()
+{
+    if (this.source.type !== 'svg') {
+        // Do nothing if source is not svg
+        return;
+    }
+
+    if (!this.realWidth || !this.realHeight || this.realWidth <= 1 || this.realHeight <= 1 ||
+        (this.realWidth === 150 && this.realHeight === 150)) {
```

Comment:
@englercj basically yes. I didn't find a neat way to determine if the SVG has size properties or not. With inline svg that can be checked but when loaded using Image API I don't know a way to access the original width and height of the svg. So now it relies on the fact that Chrome sets width and height to 150. One should anyway use pow2 textures, right?